nixops/resources/ec2_common.py

```diff
@@ -1,10 +1,14 @@
 import socket
 import getpass
+
 import nixops.util
 import nixops.resources
+from nixops.diff import Diff, Handler
 
 class EC2CommonState():
 
+    COMMON_EC2_RESERVED = ['accessKeyId', 'ec2.tags']
```

Comment:
Why is `ec2.tags` reserved?

Reply:
@edolstra oops I think I can remove that.

Reply:
@edolstra actually since we have `tags = nixops.util.attr_property("ec2.tags", {}, 'json')` in `nixops.ec2_common.EC2CommonState` I need to keep this for now as I'm inheriting from that class.